### PR TITLE
Chen2020Learning2

### PR DESCRIPTION
**Pull request by SShivang**

### CODE REVIEW

File: _summaries/chen2020learning_2.md [hunk 1]
```diff
@@ -1,19 +1,61 @@
...
+
+How well does the paper perform?
+
+![Qualitative Performance](chen2020learning_2_10.png)
+
+The paper shows that given a 48x48 path, they are able to make the best x30 resolution image.
+
+![Quantitative Performance](chen2020learning_2_11.png)
+
+They show outperformance to other methods across all datasets.
```
Comment (sritank) [inline _summaries/chen2020learning_2.md:49]: They surpass them in Out-of-distribution tasks, but for in-distribution tasks they have almost same performance

File: _summaries/chen2020learning_2.md [hunk 2]
```diff
@@ -1,19 +1,61 @@
...
+
+![Example z and x values](chen2020learning_2_4.png)
+![Deriving the image given model](chen2020learning_2_3.png)
+
+This figure, shows how the model works intuitively. We divide the image into a grid and for each grid cell we calculate a latent vector representation z. We use the vector z, associated with the grid square closest to the coordinate.
+
+![Example z and x values](chen2020learning_2_5.png)
+![Example z and x values](chen2020learning_2_6.png)
+
+The paper also makes certain optimizations to this original idea. For each of the latent vectors, they concatentate the neighboring cell latent vector to find the new latent vector. (Feature Unfolding)  This allows for a richer representation. Another optimization they make is that instead of using the closest latent vector they use the weighted average of latent vector (weighted by the distance from the coordinate x). (Local Ensemble)
```
Comment (sritank) [inline _summaries/chen2020learning_2.md:30]: Maybe you could state why they do these optimizations. The Local ensemble optimization is applied to remove discontinuities that arise when using the originally proposed function.

File: _summaries/chen2020learning_2.md [hunk 3]
```diff
@@ -1,19 +1,61 @@
...
+
+The paper does some interesting ablation testing. They try to see the effect of each of the optimizations that were discussed earlier. It doesn't seem that any of the optimizations make a significant impact on performance. Removing cell encoding seems to be the most impactful.
+
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* For image models, images have to be resized to the same size (losing information in the process)
+* Authors present a Implicit Image Function as a way to generate images given an input resolution
```
Comment (sritank) [inline _summaries/chen2020learning_2.md:60]: using an encoder-decoder architecture

File: _summaries/chen2020learning_2.md [hunk 4]
```diff
@@ -1,19 +1,61 @@
...
+* Their model delivers SOTA performance on the task
```
Comment (sritank) [inline _summaries/chen2020learning_2.md:61]: what tasks?

**Review by sritank — APPROVED** [review]
